pulp_docker/app/models.py

```diff
@@ -1,37 +1,177 @@
-"""
-Check `Plugin Writer's Guide`_ for more details.
-
-.. _Plugin Writer's Guide:
-    http://docs.pulpproject.org/en/3.0/nightly/plugins/plugin-writer/index.html
-"""
-
 from logging import getLogger
+from types import SimpleNamespace
+
+from django.db import models
 
-from django.db import models # noqa
+from pulpcore.plugin.models import Content, Remote, Publisher
 
-from pulpcore.plugin.models import Content, ContentArtifact, Remote, Publisher  # noqa
 
 logger = getLogger(__name__)
 
 
-class DockerContent(Content):
+MEDIA_TYPE = SimpleNamespace(
+    MANIFEST_V1='application/vnd.docker.distribution.manifest.v1+json',
+    MANIFEST_V2='application/vnd.docker.distribution.manifest.v2+json',
+    MANIFEST_LIST='application/vnd.docker.distribution.manifest.list.v2+json',
+    CONFIG_BLOB='application/vnd.docker.container.image.v1+json',
+    REGULAR_BLOB='application/vnd.docker.image.rootfs.diff.tar.gzip',
+    FOREIGN_BLOB='application/vnd.docker.image.rootfs.foreign.diff.tar.gzip',
+)
+
+
+class Manifest(Content):
+    """
+    A docker manifest.
+
+    This content has no artifacts.
+
+    Fields:
+        digest (models.CharField): The manifest digest.
```

Comment:
Maybe more helpful: The SHA256 (is that what this is?) if the manifest file

Reply:
as of now the sha256 is used, i would liek to leave this opened in case some other algorithms will come in place later.

Reply:
Cool, how about "The digest of the manifest file."